paper/paper.Rmd

```diff
@@ -24,140 +25,735 @@ knitr::opts_chunk$set(echo = TRUE)
 
-The `survey` package [@lumleyone, @lumleytwo, @lumleythree] is well-established and allows a user to specify a survey design, such as clustered, stratified, etc, generate summary statistics, and conduct sub-population analysis. The `DeclareDesign` package [@citeDeclareDesign] is a newer package that is based around a grammar of research design aspects including: data generation, creating treatment and control groups, and sampling. There is also an existing package called `MRP` available here: https://github.com/gelman/mrp. However, it does not appear to have been updated in some time.
+The main alternative to using `mrpkit` is for MRP to be conducted on a case-specific basis. This means that a researcher who is interested in MRP estimates writes the code needed to obtain, clean, prepare, analyze, and ultimately interpret the estimates. Again, while there is nothing inherently wrong with this approach, successfully conducting MRP requires dealing with a variety of issues. Each of these is small, but getting them wrong can have large effects on the estimates that are potentially unnoticed. It is also easy to overlook steps, or to not appropriately document them. `mrpkit` is prescriptive about the steps that must be taken when conducting MRP, and ensures that all aspects are documented.
 
-The main alternative approach is for MRP to be conducted on a case-specific basis. This means that a researcher who is interested in MRP estimates writes the code needed to obtain, clean, prepare, analyze and ultimately interpret the estimates. Again, while there is nothing inherently wrong with this approach, successfully conducting MRP requires dealing with a large number of small issues. Each of these is small in their own right but getting them wrong can have large effects that are potentially unnoticed on the estimates. It is also easy to overlook steps, or to not appropriately document them. The `MRPKit` package is prescriptive about the steps that must be taken, and ensures that all aspects are documented.
+The remainder of this paper is structured as follows: Section \@ref(usecase) discusses the core use case and workflow for MRP as implemented in `mrpkit`. Section \@ref(implementation) discusses some of the implementation issues and technical notes related to the decisions that were made. Section \@ref(example) provides an example of the package in use. Finally, Section \@ref(summary) provides a summary discussion, some cautions, and weaknesses, as well as notes about next steps.
 
 
+# Use cases and workflow {#usecase}
 
```

Comment:
IMO I think we should just keep the election example (which is heaps nice), and move some of this content to that section.